app/src/main/java/com/quran/labs/androidquran/ui/PagerActivity.java

```diff
@@ -799,12 +801,14 @@ public void onNewIntent(Intent intent) {
     }
   }
 
+  @Override
   public void jumpTo(int page) {
     Intent i = new Intent(this, PagerActivity.class);
     i.putExtra("page", page);
     onNewIntent(i);
   }
 
+  @Override
```

Comment:
Grouping is a good idea, but at the top or bottom - that changes from person to person. I'd wait for @ahmedre on that.